planning/behavior_path_planner/src/utils/goal_planner/default_fixed_goal_planner.cpp

```diff
@@ -53,11 +109,19 @@ PathWithLaneId DefaultFixedGoalPlanner::modifyPathForSmoothGoalConnection(
       refined_goal = goal;
     }
   }
-
-  auto refined_path = utils::refinePathForGoal(
-    planner_data->parameters.refine_goal_search_radius_range, M_PI * 0.5, path, refined_goal,
-    goal_lane_id);
-
+  double goal_search_radius{planner_data->parameters.refine_goal_search_radius_range};
+  const double range_reduce_by{
+    1};  // set a reasonable value, 10% - 20% of the refine_goal_search_radius_range is recommended
```

Comment:
used constexpr rather than const [https://github.com/autowarefoundation/autoware.universe/commit/08c6f53218b11f02b4c48ca47274aabd9dc67f9c](https://github.com/autowarefoundation/autoware.universe/commit/08c6f53218b11f02b4c48ca47274aabd9dc67f9c)